HW8/marian_hrychak/task_1/main.py

```
@@ -0,0 +1,4 @@
+from utils import *
+from models import *
+
+print(list(filter(lambda str: not ("__" in str), dir())))
```

Ruff (0.8.2)

Test for membership should be `not in`

Convert to `not in`

(E713)